x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/common/Failure.java

```diff
@@ -50,7 +50,7 @@ public boolean equals(Object obj) {
         }
 
         Failure other = (Failure) obj;
-        return Objects.equals(message, other.message) && Objects.equals(node, other.node);
+        return Objects.equals(node, other.node);
```

Comment:
This is done on purpose so that Failures on the same node get collapsed:
Line 1:15 ...
Line 1:15...

So far all Failures are related and it's easier to just have one overall.